nestkernel/conn_builder.cpp

```diff
@@ -386,7 +386,8 @@ nest::ConnBuilder::connect()
     && not symmetric_ && not is_symmetric() )
```

Comment:
I find this still confusing, and I fear anyone reading this in five years time will not feel enlightened. How about renaming `symmetric_` to `make_symmetric_` and turning the order of checks a little:
```C++
if ( kernel().model_manager.connector_requires_symmetric( synapse_model_ )
     and not ( is_symmetric() or make_symmetric_ ) )
```
That seems more readable to me.